src/client/chunks-api/ChunkManager.ts

```diff
@@ -61,12 +61,21 @@ class ChunkManager {
     }
   }
 
-  async invalidateChunk(chunkId: string) {
+  async invalidateChunks(chunksIds: string[] = []) {
     try {
-      const url = await this.resolveChunk(chunkId);
-      const chunkHash = this.getChunkHash(url);
-      delete this.resolveCache[chunkId];
-      await NativeModules.ChunkManager.invalidateChunk(chunkHash, chunkId, url);
+      const chunks = await Promise.all(
+        chunksIds.map(async (chunkId) => {
+          const url = await this.resolveChunk(chunkId);
+          const chunkHash = this.getChunkHash(url);
+          delete this.resolveCache[chunkId];
+
+          return {
```

Comment:
@zamotany is there a possibility that we will add more data here (ex. chunkId)? If not I can just have a list of hashes instead of objects and send it to native